examples/tests/test_register_file.py

```diff
@@ -0,0 +1,27 @@
+import random
+import pytest
+
+from ..register_file import RegisterFile
```

Comment:
any chance there's a better way to do this?

Reply:
I'm going to change the examples format to be in the form a jupyter notebook, which would put the test code in the same "file" and avoid the import issue (also making it easier to associate the test/design code in one place).